docs: plugin authoring workflow overview pages (#20847)
gatsbyjs/gatsby: examples/using-multiple-local-plugins/README.md

# Using Gatsby with Local Plugins

This is an example repository demonstrating different ways to load plugins in a Gatsby site.

## Running the Example

The actual Gatsby site to run is in the `gatsby-site-using-local-plugins` folder.

> Navigate into the example site from the root of this example with this command:

Reviewer: ```suggestion
Navigate into the `gatsby-site-using-local-plugins` project directory with this command:
```

```sh
cd gatsby-site-using-local-plugins
```

You'll need to install dependencies for the site by running:

```sh
npm install
```

Then run `gatsby develop`:

```sh
gatsby develop
```

> You should then see the output listed below. Showing how the code for each plugin is run sequentially thanks to the Node API implemented.

Reviewer: ```suggestion
In your command line output, you should then see the text listed below. This text is showing how the code for each plugin is run sequentially thanks to the Node API implemented.
```
